### ISSUE TITLE
[Bug Report][2.3.10]  the NVDA screen reader is not announcing the error messages generated by v-text-feild

### ISSUE THREAD

**Issue by gavinwang-au** [vuetifyjs/vuetify#15554]
### Environment
**Vuetify Version:** 2.3.10
**Vue Version:** 2.6.10
**Browsers:** Chrome 103.0.0.0
**OS:** Mac OS 10.15.7

### Steps to reproduce
1. input the wrong data to the email field
2. try to use Tab to navigate to the email input field
3. the error message 'invalid e-mail' is not announceded by the NVDA screen reader. 

### Expected Behavior
The NVDA screen reader should announce the error message correctly 

### Actual Behavior
the NVDA screen reader is not announcing the error message

### Reproduction Link
<a href="https://codepen.io/gavinwang-au/pen/ZExvxmN?editors=101" target="_blank">https://codepen.io/gavinwang-au/pen/ZExvxmN?editors=101</a>

**Comment by KaelWD** [comment]
Duplicate of #13874
